build.gradle

```diff
@@ -14,7 +14,13 @@ buildscript {
     }
 }
 
+
 allprojects {
+
+    String osName = System.getProperty("os.name").toLowerCase();
+    if (osName.contains("windows")) {
+        buildDir = "C:/tmp/${rootProject.name}/${project.name}"
+    }
```

Comment:
I can see why you'd want to do this with CI pipelines and things but why have you put it in here? What's it achieving? Futureproofing?